src/robotide/ui/tagdialogs.py

```diff
@@ -100,6 +102,15 @@ def _make_bindings(self):
         self._tags_list.Bind(wx.EVT_LIST_ITEM_RIGHT_CLICK, self.OnRightClick)
         self._tags_list.Bind(wx.EVT_LIST_COL_CLICK, self.OnColClick)
 
+    def _tag_name_for_sort(self, tag_name):
+        # Make tag_name lowercase for the sorting algorithm only
+        tag_name = tag_name.lower()
+        # Compare numbers as numeric values
+        # m = re.search(r"([^0-9]*)([0-9]*)", tag_name)
+        m = re.search(r"(\w*)([0-9]*)", tag_name)
+        g = m.groups()
```

Comment:
This code is still buggy. It ignores everything after the first group of digits like it did earlier. In other words, order of tags like `1a`, `1b` and `1c` is undefined. I'm not sure what's the best way to fix that.